### ISSUE TITLE
Commit the code that has been modified on prologin@rosa:~/site

### ISSUE THREAD

**Issue by audebert** [prologin/concours-site#47]
> ###### Originally posted 2016-01-11 00:53 by [halfr](https://bitbucket.org/halfr/) on Bitbucket

```
#!shell

prologin@rosa:~/site$ git status
On branch master
Your branch is up-to-date with 'origin/master'.
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

        modified:   prologin/mailing/management/commands/remove_bounces.py
        modified:   prologin/mailing/templates/mailing/end_qualifications.body.text.html
        modified:   prologin/mailing/templates/mailing/end_qualifications.subject.html

```

**Comment by audebert** [comment]
> ###### Originally posted 2016-01-13 00:33 by [halfr](https://bitbucket.org/halfr/) on Bitbucket

mailing: remove_bounces: fix (resolve #47)


→ <<cset 2e8dd34b0f69>>

**Comment by zopieux** [comment]
> ###### Originally posted 2016-01-13 22:09 by [Zopieux](https://bitbucket.org/Zopieux/) on Bitbucket

Many non-trivial modifications that are not commited.

```text
	modified:   prologin/documents/templates/documents/convocation-regionale.tex
	modified:   prologin/documents/templates/documents/droit-image-regionale.tex
	modified:   prologin/mailing/management/commands/mailing.py
	modified:   prologin/mailing/templates/mailing/semifinal_qualified.body.text.html
```

Please cleanup this mess ASAP.

**Comment by audebert** [comment]
> ###### Originally posted 2016-01-14 09:53 by [halfr](https://bitbucket.org/halfr/) on Bitbucket

Closed by a75c449cb909f8751af0ec39fc1bdf5d86c3c420
